homeassistant/scripts/check_config.py

```diff
@@ -105,26 +106,26 @@ def run(script_args: List) -> int:
 
     if len(res['except']) > 0:
         print(color('bold_white', 'Failed config'))
-        for domain, config in res['except'].items():
+        for domain, config in sorted(res['except'].items()):
```

Comment:
By using `sorted` you're overriding the order of the items in the config, is this on purpose?